static/app/views/performance/http/httpLandingPage.tsx

```diff
@@ -0,0 +1,175 @@
+import React, {Fragment} from 'react';
```

Comment:
importing react is not recommended

```suggestion
import {Fragment} from 'react';
```

Reply:
Our linting forces `React.Fragment` over `Fragment`, so I have to include the import